maggy/tests/test_review_pipeline.py

```
@@ -54,6 +54,52 @@ def test_cache_discount_applied(self):
 
 
+class TestCouncilRoster:
+    def test_terra_in_roster(self):
+        from maggy.review.config import ROSTER
+        names = {r[0] for r in ROSTER}
+        assert "Shannon" in names, "GPT-5.6 Terra ('Shannon') must be on the council"
+
+    def test_terra_entry_shape(self):
+        from maggy.review.config import ROSTER
+        entry = next(r for r in ROSTER if r[0] == "Shannon")
+        name, factory, label, need, tools = entry
```

Ruff (0.15.20)

[warning] Unpacked variable `name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)